### ISSUE TITLE
Hi, is there a way to use this in Platformio without Arduino plarform?

### ISSUE THREAD

**Issue by shablam** [Bodmer/TFT_eSPI#2098]
HI, i want to use this but in Platformio i am using wizio-pico. framework is bare metal. Below are the basic flags for ref. 

I try to add the header <TFT_eSPI.h> and it causes lot of wrrors in the header file and elsewhere such as "variable "uint16_t" is not a type name".
it looks for the definition of uint16_t in the tft_espi.h file rather than the _stndin file. This is in C so not sure if there is a C++ / C issue. Maybe you can help. 

`
platform = wizio-pico
board = raspberry-pi-pico
framework = baremetal
build_flags = 
	
	-D PICO_STDIO_UART
	-D PICO_USB_HOST
`

**Comment by Bodmer** [comment]
Yes, users have done this. You must load the Arduino support package into PlatformIO, there are online tutorials for this I I do not provide support for that aspect..
